docs/reference/operator_parameters.md

```diff
@@ -161,6 +161,13 @@ configuration they are grouped under the `kubernetes` key.
   replaced by the cluster name. Only the `{cluster}` placeholders is allowed in
   the template.
 
+* **enable_pod_disruption_budget**
+  if disabled `MinAvailable` in the PDB's spec will be set to `0`. That means
+  the PDB will get created anyway. The PDB is also relaxed when scaling down the
+  Postgres cluster to `"numberOfInstances": 0`. When scaling up or enabling this
```

Comment:
Why do we impl. the downscale part without the upscale working properly? Whats the win of having pdb at 0 for 0 pods?

Reply:
Pdb at 0 has been suggested in #384. I found out that PDB did not get synced on UPDATE event. Only on SYNC. Now it will be set to 1 as soon as the cluster is scaling up.